### PR TITLE
A new blog post to share GitHub Insight Report 2020 with CHAOSS

### PR DESCRIPTION
**Pull request by xiaoya-Esther**

### CODE REVIEW

File: Community/News/20210407_GitHub_Analysis_Report.md [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+![2021040603](./images/2021040603.jpg)
+
+
+
+![2021040604](./images/2021040604.jpg)
+
+
+
+![2021040605](./images/2021040605.jpg)
+
```
Comment (GeorgLink) [inline Community/News/20210407_GitHub_Analysis_Report.md:36]: Please remove `./` from the relative paths

File: Community/News/20210407_GitHub_Analysis_Report.md [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+**GitHub**, the world's largest open source collaboration platform, produced a massive amount of developer behavior data.
+
+We obtained **860 million** event logs generated in 2020 and analyzed **14.54 million** active developers and **54.21 million** active projects.
+
+What have we learned?
+
+Please read our [GitHub 2020 Digital Insight Report](http://oss.x-lab.info/github-insight-report-2020-en.pdf).
+
+![2021040601](/Users/xiaoya/Documents/GitHub/website/Community/News/images/2021040601.jpg)
```
Comment (GeorgLink) [inline Community/News/20210407_GitHub_Analysis_Report.md:17]: Please make path relative to the GitHub repo

**Review by GeorgLink — CHANGES_REQUESTED** [review]
Thanks @xiaoya-Esther 

Please change the way images are loaded

File: Community/News/20210407_GitHub_Analysis_Report.md (hunk 2 above)
Comment (xiaoya-Esther) [reply Community/News/20210407_GitHub_Analysis_Report.md]: Thanks Georg. I've made those changes.

Comment (GeorgLink) [reply Community/News/20210407_GitHub_Analysis_Report.md]: Awesome, thanks!

**Pull request merged**